packages/runtime/src/sheet.tsx

```diff
@@ -22,7 +22,7 @@ function createStyleElement(opts: StyleSheetOpts): HTMLStyleElement {
  *
  * @param opts StyleSheetOpts
  */
-export const createStyleSheet = (opts: StyleSheetOpts) => {
+export default function createStyleSheet(opts: StyleSheetOpts) {
```

Comment:
nit: hehe. I thought you like named export 😛. Probably if we are using default export, it should match filename right 🤔?

Reply:
only did this to reduce file size down haha - it reduces it a few bytes

Reply:
😱 wooo, never guessed that. Damn. Nice.

Reply:
And btw how it reduced the size 🤔 thinking

Reply:
when minified default imports reduce down to `a` - something small

named imports cant be reduced it seems (you probably can but requires more complex minification)

Reply:
ohh right. hehe. I totally forgot minification here 😛 ahh another drawback of using named export then hehehehe 😛